docs/GETTING-STARTED.md

```diff
@@ -0,0 +1,424 @@
+# Getting started with Network Service Mesh
+
+This document illustrates the procedure to install and start network service mesh from a bare bones system.`
+
+## Table of Contents
+1. [Introduction](#"Introduction")
+2. [Set up Virtualization](#"Nested")
+3. [Install on Ubuntu](#"Ubuntu")
+4. [Install on Fedora.](#"Fedora")
+5. [Install pre-requisites](#"prereq")
+6. [Install docker](#"docker")
+7. [Install docker CE](#"dockerCE")
+8. [Install kubernetes](#kube)
+9. [Install virt and kvm](#virt)
+10. [Install kubectl](#kubectl)
+11. [Install golang](#golang)
+12. [Install minikube ](#minikube)
+13. [Install kvm2 driver](#kvm2)
+14. [Install protobuf](#proto)
+15. [Install NSM](#NSM)
+16. [Verify NSM](#NSMverify)
+17. [Manual steps](#Manual)
+18. [TODO](#TODO)
+19. [Credits](#credit)
+
+## Introduction
+
+For a working example we will launch a VM running kubernetes, and all required packages as well as network service mesh itself. The exam;e starts with launching a VM under Qemu/KVM. You may of course choosse a different hypervisor but those procedures haven't been verified in this document.
```

Comment:
This example doesn't run NSM. Instead, it guides the user building a development environment and compiling NSM. 

exam;e => example
choosse => choose